node-operators/network-node/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Network Node
```

Comment:
```suggestion
title: Run a Node on a Tanssi-Powered Network
```

This title is for SEO purposes, so let's try to add a little bit more context in these titles when possible. But one thing to keep in mind is that these titles can be no more than 60 characters, and ` | Tanssi Docs` is appended to the end of the titles automatically. So you really have like 46 characters to work with